crates/sargon/src/gateway_api/models/types/request/state/account_page.rs

```diff
@@ -0,0 +1,29 @@
+use crate::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct AccountPageRequest {
+    /// Bech32m-encoded human readable version of the address.
+    pub(crate) account_address: AccountAddress,
+
+    /// This cursor allows forward pagination, by providing the cursor from the previous request.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub(crate) cursor: Option<String>,
```

Comment:
We should also have the `at_ledger_state`, very important that we use the proper ledger state for pagination.